infrastructure/core/variables.tf

```diff
@@ -99,11 +99,21 @@ variable "accesses_real_data" {
   description = "Does this deployment access real data? I.e. is this a staging/production environment?"
 }
 
-variable "alert_recipients" {
-  description = "List of recipients to receive alerts"
-  type = list(object({
-    name  = string
-    email = string
-  }))
-  default = []
+variable "monitoring" {
+  description = "Monitoring block"
+  type = object({
+    alert_recipients = list(object({ # List of recipients to receive alerts
```

Comment:
Should alert recipients be able to be set without setting network watcher and vice versa? If so, you'll need to mark these both as optional and provide individual defaults